web/Dockerfile

```diff
@@ -26,6 +27,7 @@ RUN set -ex \
 		dj-database-url \
 		mysqlclient \
 		typing \
+	&& python3 -m pip install git+https://github.com/notanumber/xapian-haystack.git \
```

Comment:
Not a big fan of installing packages directly from Github, is this package not published in PyPI?

Reply:
I went by the documentation on that repo: https://github.com/notanumber/xapian-haystack#installation. It exists on PyPI, so I'll update the routine to use https://pypi.org/project/xapian-haystack/ instead.